draft-pauly-intarea-proxy-config-pvd.md

```diff
@@ -275,14 +309,40 @@ proxies, can only be safely used when fetched over a secure TLS-protected connec
 and the client has validated that that the hostname of the proxy, the identifier of
 the PvD, and the validated hostname identity on the certificate all match.
 
-# IANA Considerations {#iana}
+# IANA Considerations
+
+## New PvD Additional Information key {#proxies-key-iana}
 
 This document registers a new key in the "Additional Information PvD Keys" registry.
 
 JSON Key: proxies
 
-Description: Array of proxy URIs associated with this PvD
+Description: Array of proxy dictionaries associated with this PvD
+
+Type: Array of dictionaries
+
+Example: [ {
+  "protocol": "connect",
+  "uri": "https://proxy.example.com"
+} ]
+
+## New PvD Proxy Information Registry {#proxy-info-iana}
+
+IANA is requested to create a new registry "Proxy Information PvD Keys", within the "Provisioning Domains (PvDs)" registry page.
+This new registry reserves JSON keys for use in sub-dictionaries under the `proxies` key.
+The initial contents of this registry are given in {{proxy-enumeration}}.
+
+The status of a key as mandatory or optional is intentionally not denoted in the table to allow for
+flexibility in future use cases. Any new assignments of keys will be considered as optional for the purpose of the mechanism described in this document.
+
+New assignments in the "Proxy Information PvD Keys" registry will be administered by IANA through Expert Review {{!RFC8126}}. Experts are
+requested to ensure that defined keys do not overlap in names or semantics.
+
+## New PvD Proxy Protocol Registry {#proxy-protocol-iana}
 
-Type: Array of strings
+IANA is requested to create a new registry "Proxy Protocol PvD Values", within the "Provisioning Domains (PvDs)" registry page.
+This new registry reserves JSON values for the `protocol` key in `proxies` sub-dictionaries.
+The initial contents of this registry are given in {{proxy-enumeration}}.
 
-Example: ["https://proxy.example.com", "https://proxy.example.com/masque{?target_host,target_port}"]
+New assignments in the "Proxy Protocol PvD Values" registry will be administered by IANA through Specification Required {{!RFC8126}}.
```

Comment:
Can we just use Expert Review? This registry has infinite space so restricting allocation will only lead to squatting

Reply:
Since we need a reference document to point to in the table, it felt like a specification required one?

Reply:
Can't we say that this field is allowed to be empty? That's what many other registries do

Reply:
e.g. https://www.iana.org/assignments/masque/masque.xhtml

Reply:
Allowing it to be empty now